tsconfig.module.json

```diff
@@ -9,7 +9,22 @@
     "noEmit": false,
     // "emitDeclarationOnly": true,
     // Uncomment to debug
-    // "listEmittedFiles": true
+    // "listEmittedFiles": true,
+    "plugins": [
+      {
+        "transform": "ocular-dev-tools/ts-transform-version-inline"
+      },
+      {
+        "transform": "ocular-dev-tools/ts-transform-append-extension",
+        "extensions": [".js"],
```

Comment:
I can see there are files named `glsl.ts` in this repo. You will need to add `.glsl.js` here to process imports from `./xxx.glsl`.

Reply:
Good point. I haven't been able to get these to take effect yet. I will continue in a separate PR.